js-miniapp-bridge/src/bridge.js

```diff
@@ -96,6 +97,21 @@ MiniAppBridge.getUniqueId = function() {
     );
   });
 };
+
+/**
+ * Associating requestPermission function to MiniAppBridge object
+ * @param {String} requestType Type of permission that is requested. For eg., location
+ */
+MiniAppBridge.requestPermission = function(requestType) {
+  return new Promise(function(resolve, reject) {
+    return MiniAppBridge.exec(
+      "requestPermission",
+      requestType,
```

Comment:
Will it make more sense to pass a JSON object as the params? It would let the `exec` function be more generic and then we can more easily add more params in the future if needed. So here you could pass something like `{permission: 'some_permission'}`.

Reply:
Sure, I even thought of doing the same.